main.tf

```diff
@@ -6,4 +6,9 @@ resource "azurerm_resource_group" "rg-dhondhu" {
 resource "azurerm_resource_group" "rg-jackie" {
   name     = "rg-jackie"
   location = "West Europe"
+}
+
+resource "azurerm_resource_group" "rg-rahul" {
+  name     = "rg-rahul"
+  location = "CentralIndia"
```

Comment:
yeh galat hai. location sahi karo location us ki honi chayee

Reply:
sahi s check karo aap

Reply:
Done sir Location updated to west us

Reply:
Ab theek hai